showkase/build.gradle

```diff
@@ -21,11 +21,11 @@ kapt {
 }
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 30
 
     defaultConfig {
-        minSdkVersion 21
-        targetSdkVersion 29
+        minSdkVersion 26
```

Comment:
I know this is still work in progress and you added this only because of this build error

```
/Users/runner/work/Showkase/Showkase/showkase-processor-testing/libs/rt.jar: D8: com.android.tools.r8.internal.h1: MethodHandle.invoke and MethodHandle.invokeExact are only supported starting with Android O (--min-api 26)
```

But I'm keen on not updating this minSdkVersion. I think it's important to keep the library accessible and 21 is a reasonable `minSdkVersion`. There's still a way to use this if the consumers of this library add an override using `tools:overrideLibrary` but I would like to not have them do that.